src/components/controls/Card.js

```diff
@@ -201,11 +200,11 @@ export const Card = ({
     }
   }
 
-  function renderRow(row) {
+  function renderRow(row, rowIdx) {
     return (
-      <div className="card-row" key={hash(row)}>
-        {row.map((field) => (
-          <span key={hash(field)}>{renderField(field)}</span>
+      <div className="card-row" key={rowIdx}>
```

Comment:
Assuming rows don't reorder while the app is running, using index should be fine. Same for fields.